circle.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+
+jobs:
+  swift4:
+    docker:
+      - image: swift:4.2
```

Comment:
make that `swift:5.0`, NIO 1 just fine with Swift 5. But it should become NIO2 anyway.

Reply:
I already have Swift 5 in there too, this was more for backwards compatability to 4.2. But if this is NIO 2, that will not be there anyway, right?

Reply:
Correct, NIO2 is only 5+

Reply:
I went with the suggestion from @weissi to use `1.0 ..< 3.0` for NIO :)